app/models/nextUpdates/NextUpdatesResponseModel.scala

```diff
@@ -136,3 +131,27 @@ object NextUpdatesModel {
 object NextUpdatesErrorModel {
   implicit val format: Format[NextUpdatesErrorModel] = Json.format[NextUpdatesErrorModel]
 }
+
+sealed trait ObligationStatus
```

Comment:
looks good thanks James.
Can we copy these changes to the BE repo? We should have similar models FE/BE.

Reply:
yes of course Joe, will discuss with @ian-hmrc and create tickets appropriately